src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx

```diff
@@ -185,10 +170,6 @@ const debouncedBroadcastUserIsTyping = lodashDebounce((reportID: string) => {
 
 const willBlurTextInputOnTapOutside = willBlurTextInputOnTapOutsideFunc();
 
-// We want consistent auto focus behavior on input between native and mWeb so we have some auto focus management code that will
-// prevent auto focus on existing chat for mobile device
-const shouldFocusInputOnScreenFocus = canFocusInputOnScreenFocus();
-
```

Comment:
Why this was deleted?

Reply:
Same question. 

After rechecking. This is added https://github.com/Expensify/App/pull/18699 and now we change the expectation. So this should be removed to align with new expect